tests/test_scheduler.py

```diff
@@ -66,18 +70,19 @@ def tearDown(self):
 
     def test_add_experiment(self):
         app = scheduler.SchedulerApp(name="name")
-        with mock.patch.object(app.schedulerFrame.model, "experimentQueue") as mockedQueue:
+        with mock.patch.object(app.schedulerFrame.model, "experimentQueue") as mocked_queue:
             data = [ExperimentInfo(str(i), {"rid": i, "priority": 10 - i}) for i in range(10)]
             for info in data:
                 app.addExperiment(info)
-                mockedQueue.append.assert_called_with(info)
+                mocked_queue.append.assert_called_with(info)
+                assert mocked_queue.sort.called
```

Comment:
Please use `mocked_queue.sort.assert_called_once()` or something like that, instead of `assert`.